bundles/core/src/main/java/com/adobe/cq/commerce/core/components/client/DeniedHttpHeaders.java

```diff
@@ -0,0 +1,77 @@
+/*
+ *  Copyright 2021 Adobe. All rights reserved.
+ *
+ *   This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+package com.adobe.cq.commerce.core.components.client;
+
+import java.util.Set;
+
+import com.google.common.collect.ImmutableSet;
+
+public interface DeniedHttpHeaders {
```

Comment:
I wonder if we should consider reusing this from the add-on?

Reply:
The add-on does not really provide any Java API yet, only the GraphQL client is exposed. However we already have an add-on SDK API released and it can be added there as well.

Reply:
But even if we do this, it won't be released until after the components are released, so the only solution is to do this for 2.0 (next release)

Reply:
Yes I agree, timing will be complicated.